code/game/objects/items.dm

```diff
@@ -1568,5 +1569,9 @@ var/global/list/image/blood_overlays = list()
 		var/tool_sound = pick(toolsounds)
 		playsound(A, tool_sound, volume, TRUE, vary)
 
+/obj/item/proc/playsurgerysound(atom/A, var/volume = 75, vary = TRUE, extrarange = null)
+	if(A && surgerysound)
+		playsound(A, surgerysound, volume, TRUE, vary)
```

Comment:
The arguments you're passing to `playsound` are confusing.
Currently `playsound` is defined as
`/proc/playsound(var/atom/source, soundin, vol as num, vary = 0, extrarange as num, ...)` .
You're passing `TRUE` unconditionally as the fourth argument, which is `vary`, ignoring the `vary` parameter to `playsurgerysound`.
Then you pass `vary` as the fifth argument, but `playsound`' expects the fifth argument to be the value of `extrarange`.

Sort this out, then make sure it's still working as you intended.
I'd just remove both `vary` and `extrarange` from `playsurgerysound`'s definition, and have it call `playsound` with `vary = TRUE`.

Reply:
DONE.